src/main/java/io/cdap/plugin/gcp/publisher/source/PubSubSubscriberUtil.java

```diff
@@ -34,6 +48,21 @@ public final class PubSubSubscriberUtil {
 
   protected static final Logger LOG = LoggerFactory.getLogger(PubSubSubscriberUtil.class);
 
+  // Retryable status codes. These need to be handled in case Pub/Sub throws a StatusRuntimeException.
+  private static final int RESOURCE_EXHAUSTED = StatusCode.Code.RESOURCE_EXHAUSTED.getHttpStatusCode();
+  private static final int CANCELLED = StatusCode.Code.CANCELLED.getHttpStatusCode();
+  private static final int INTERNAL = StatusCode.Code.INTERNAL.getHttpStatusCode();
+  private static final int UNAVAILABLE = StatusCode.Code.UNAVAILABLE.getHttpStatusCode();
+  private static final int DEADLINE_EXCEEDED = StatusCode.Code.DEADLINE_EXCEEDED.getHttpStatusCode();
+
+  private static final Set<Integer> RETRYABLE_STATUS_CODES = Collections.unmodifiableSet(new HashSet<Integer>() {{
```

Comment:
nit: That's a weird construct with anonymous classes. I see that it was like this before, that's why it's `nit`

Reply:
Changed.